core/modules/drr.cc

```diff
@@ -260,14 +261,14 @@ DRR::Flow* DRR::GetNextFlow(int* err) {
   return f;
 }
 
-uint32_t DRR::GetNextPackets(bess::PacketBatch* batch, Flow* f, int* err) {
+uint32_t DRR::GetNextPackets(bess::PacketBatch *batch, Flow *f, int *err) {
   uint32_t total_bytes = 0;
-  bess::Packet* pkt;
+  bess::Packet *pkt;
```

Comment:
No... I don't know why yet, I haven't investigated much..

Reply:
clang-format is fine with either style, but it tries to make all consistent. The current majority style is chosen, so it will depend on which one is majority.